src/components/img/Img.vue

```diff
@@ -114,6 +114,16 @@ export default {
 
             return null;
         },
+        computedSrc() {
+            if (this.src.indexOf('https://www.visitscotland.com/binaries/content/gallery/visitscotland/cms-images/') === 0) {
+                return this.src.replace(
+                    'https://www.visitscotland.com/binaries/content/gallery/visitscotland/cms-images/',
+                    'https://d2mq8p11a67q50.cloudfront.net/',
+                );
+            }
+
```

Comment:
This is very ugly. Currently the image scaler is set up to retrieve any path it receives (e.g. [scalerurl]/images/2020/orkney.png) relative to the cms images path on .com, so this ensures it will be able to handle anything it receives. The possible alternative is to pass the entire path of the image to the scaler as a url parameter instead, then it could retrieve any image from any site in theory and this would be a bit cleaner, but whether that's appropriate depends on how exactly we're going to use it.

This instance of the scaler is also just a playground one, a live one would need to have it's permissions and guardrails checked and I imagine we'd pass the url via .env.